.github/workflows/test.yml

```diff
@@ -36,16 +36,10 @@ jobs:
     needs: lint
     strategy:
       matrix:
-        python: ['3.8', '3.9']
-        django: ['3.2']
-        wagtail: ['4.1', '4.2']
-        include:
-          - python: '3.10'
-            django: '4.2'
-            wagtail: '5.1'
-          - python: '3.11'
-            django: '4.2'
-            wagtail: '5.1'
+        python: ['3.11', '3.12']
+        django: ['3.2', '4.2']
```

Comment:
we went with Django >= 4.2, Wagtail 5.2 in https://github.com/wagtail/wagtail-vector-index/blob/main/pyproject.toml#L28-L29. Let's do the same here